src/partials/_head.hbs

```diff
@@ -9,4 +9,5 @@
 
 	grunticon(["/icons/icons.data.svg.css", "/icons/icons.data.png.css", "/icons/icons.fallback.css"], grunticon.svgLoadedCallback);
 </script>
+<script src="/dist/index.js"></script>
```

Comment:
It would matter if you're making DOM, which it looks like you are. Looking through your source, I don't see jQuery anywhere, but your `index.js` is using a jQuery plugin no?